src/order/cart/serviceOption/order-cart-serviceOption.v6.service.js

```diff
@@ -1,3 +1,8 @@
+/**
+ *  @deprecated
```

Comment:
I think it can be totally removed.
Not only document it as deprecated

Reply:
Okay but we need a major version then :)

Reply:
You're wright it's too dangerous. This solution is the best for the moment and we need to find progressively where the other service is used before remove it.